.github/workflows/label_prs.yaml

```diff
@@ -26,20 +26,36 @@ jobs:
             documentation:
               - '*.md'
               - 'docs/**'
+              - 'examples/**'
             explorer:
               - 'explorer/**'
             infrastructure:
-              - 'BUILD'
-              - 'WORKSPACE'
+              - '*.bzl'
+              - '*.cfg'
+              - '*.toml'
               - '.*'
               - '.*/**'
+              - 'BUILD'
+              - 'MODULE.*'
+              - 'WORKSPACE'
               - 'bazel/**'
+              - 'common/**'
+              - 'github_tools/**'
+              - 'installers/**'
+              - 'language_server/**'
```

Comment:
Before, the infrastructure cluster was really low-level stuff. While common and testing could be in the middle somewhere, I think the language server and utils are very high level: they're focused on IDE support. I would suggest a "misc" label or similar for most of these changes, including third_party (which is not really infrastructure, e.g. with your boost_unordered support that's explicitly for toolchain benchmarks).

Reply:
I'm happy breaking things up more if we can come up with good category names?

I'd like to avoid "misc" as I'd like to keep things fairly descriptive, and it seems good to highlight changes to common code or "infrastructure" of any form... We don have a catch-all bucket for other changes, which I should probably move above the bot changes.

But ideally, I'd like to find more helpful descriptive categories for as much as we can. Happy to add "IDE and editor tooling" or some better phrasing for `utils/` and `language_server/` as I agree those are quite different. Any good ideas for `common/`? Seems really good to include, but not sure the right phrasing. (I'm less worried about `third_party/` as there just isn't much there, and I'll move the examples part to be documentation like the top-level examples are.)

Reply:
Perhaps a simple "third-party" label for third_party changes? For utils and language_server, I don't have a specific suggestion, because everything I come down to is essentially "misc". I mean, there's also "utils", but that's basically "misc" (although maybe "language_server" should've been under "utils" and then "utils" would make more sense)

Reply:
How about something like this:

* `tooling`: :hammer_and_wrench: (stuff that Carbon developers use for interacting with Carbon code)

              - 'language_server/**'
              - 'utils/**'

* `infrastructure`: :gear: (stuff that we use to maintain the repository etc.)

              - 'github_tools/**'
              - 'installers/**'
              - 'proposal/scripts/**'
              - 'scripts/**'

* (not sure of the best name) `foundation` / `common` / `support`: :jigsaw: (stuff that we link into the toolchain that's not toolchain-specific)

              - 'testing/**'
              - 'common/**'
              - 'third_party/**'

(Maybe we should split `testing` and `common` out from `third_party`, based on who maintains them. I'm neutral on that.)

Reply:
We're already using :hammer_and_wrench: for the toolchain but could switch that to :chains: maybe?

Reply:
After more discussion on Discord, I think I've implemented the direction folks liked, PTAL!